### PR TITLE
ETHERC20 Payment addresses section in owner tools

### PR DESCRIPTION
**Pull request by peripheralist**
## What does this PR do and why?

- Makes payment addresses viewable in the owner tools drawer
- Unifies "payment address" language throughout
- Removes "write this down or you'll lose it" message from confirmation modal after deploying a payment address

## Screenshots or screen recordings

image
image

View when clicking "X deployed payment addresses" button:
image

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [x] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by tomquirk** [comment]
@wraeth-eth could you please take over the review here? 🙏

File: src/components/v2/V2Project/ProjectPayersModal.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
+import { CheckCircleOutlined, CloseCircleOutlined } from '@ant-design/icons'
+import { Modal, Space } from 'antd'
+import FormattedAddress from 'components/FormattedAddress'
+import RichNote from 'components/RichNote'
+import { ThemeContext } from 'contexts/themeContext'
+import { ETHERC20ProjectPayer } from 'models/subgraph-entities/v2/eth-erc20-project-payer'
+import { useContext } from 'react'
+
+export default function ProjectPayersModal({
```
Comment (wraeth-eth) [inline src/components/v2/V2Project/ProjectPayersModal.tsx:9]: Should do a review on this function for where Trans/t is needed

File: src/components/Project/ProjectToolsDrawer/ProjectPayersSection.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    <section>
+      <h3>
+        <Trans>ETH-ERC20 Payment addresses</Trans>
+      </h3>
+
+      <Space direction="vertical" size="middle" style={{ width: '100%' }}>
+        <MinimalCollapse header="More information">
+          <p>
+            ETH-ERC20 Payment addresses are{' '}
+            <a
```
Comment (wraeth-eth) [inline src/components/Project/ProjectToolsDrawer/ProjectPayersSection.tsx:29]: Recommend using <ExternalLink />

**Review by wraeth-eth — CHANGES_REQUESTED** [review]
Looks good Peri! Just a couple of minor comments surrounding translations, and recommendation of ExternalLink

File: src/components/Project/ProjectToolsDrawer/ProjectPayersSection.tsx (hunk 2 above)
Comment (peripheralist) [reply src/components/Project/ProjectToolsDrawer/ProjectPayersSection.tsx]: ...ExternalLink? 😏

**Comment by peripheralist** [comment]
thank you @wraeth-eth! i really need a linter to remind me to add <Trans> 🤦

File: src/components/v2/V2Project/ProjectPayersModal.tsx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+      {projectPayers ? (
+        <div
+          style={{
+            color: colors.text.tertiary,
+            padding: gap,
+            textAlign: 'center',
+          }}
+        >
+          {projectPayers.length
```
[the remark this answers is not in the document]
Comment (tomquirk) [reply src/components/v2/V2Project/ProjectPayersModal.tsx]: dropping by to say that `plural` is useful/preferred here:

```js
import { plural } from '@lingui/macro'

plural(projectPayers.length, {
    one: '# payment address',
    other: '# payment addresses',
})
```

**Comment by peripheralist** [comment]
finally solved these i18n issues! thanks @aeolianeth for pointing out the filenames, must have been a merge error. cc @wraeth-eth

**Comment by nnnnicholas** [comment]
exciting!

**Comment by tomquirk** [comment]
@wraeth-eth could you give this a final review pls?

File: src/components/Project/ProjectToolsDrawer/ProjectPayersSection.tsx (hunk 2 above)
Comment (wraeth-eth) [reply src/components/Project/ProjectToolsDrawer/ProjectPayersSection.tsx]: lol yes - sorry. No idea what happened there

**Review by wraeth-eth — APPROVED** [review]

**Pull request merged**